rfs6-errors.json

```diff
@@ -2,25 +2,31 @@
   {
     "id" : "all",
     "name" : "All packages",
-    "total" : 3396,
+    "total" : 3395,
```

Comment:
I've had to regenerate the package lists in order to be able to match on package urls instead of id. From the last commit date I gathered we ran these on dev around the time we made the third run, so I've now based them on the snapshot we took at the time, i.e. these are stable now.

That also means the `all` list has very slightly changed retrospectively, but everything is (and was) consistent.

Reply:
We have only one hard-coded reference to this number, [here](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/11f822a2fed79b0dc09dfb608b667abb239f9f54/Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show%2BView.swift#L124). I [pushed an update](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/blob/11f822a2fed79b0dc09dfb608b667abb239f9f54/Sources/App/Views/ReadyForSwift6/ReadyForSwift6Show%2BView.swift#L124) to the `run-06-results` to match this new number.